package.json

```diff
@@ -5,11 +5,12 @@
   "private": true,
   "main": "",
   "scripts": {
-    "__eslintTheme": "eslint wp-content/themes/inf_theme/skin/assets/",
-    "__stylelintTheme": "stylelint \"wp-content/themes/inf_theme/**/*.scss\" --syntax scss",
-    "precommitCss": "npm run __stylelintTheme",
-    "precommitJs": "npm run __eslintTheme",
-    "precommit": "npm run precommitJs && npm run precommitCss && bash bin/precommit.sh",
+    "__eslintTheme": "eslint skin/assets/**/*.js",
+    "__stylelintTheme": "stylelint skin/assets/**/*.scss",
+    "lintStyle": "npm run __stylelintTheme",
+    "lintJs": "npm run __eslintTheme",
+    "lint": "npm run lintJs && npm run lintStyle && bash bin/lintPhp.sh",
```

Comment:
Can't we remove the `npm run lintJs && npm run lintStyle` and just use `npm run __eslintTheme && npm run __stylelintTheme `? Seems like an overhead. Also, we could just rename the first two (line 8 and 9) to `lintJS` and `lintStyle`, which would remove the need for line 10 and 11.

Reply:
@dingo-d Well I'd run `npm run lintJs` rather than running `run __eslintTheme`, it just looks more friendly :D So renaming the __ ones (lines 8 & 9) makes more sense imo.